packages/experimental/Badge/src/Badge.types.ts

```diff
@@ -10,40 +10,17 @@ export type BadgeAppearance = 'filled' | 'outline' | 'tint' | 'ghost' | 'filledI
 export type BadgeShape = 'rounded' | 'circular' | 'square';
 
 export interface BadgeTokens extends LayoutTokens, FontTokens, IBorderTokens, IShadowTokens, IColorTokens {
-  /**
-   * The icon color.
-   */
   iconColor?: ColorValue;
-
-  /**
-   * The icon color when hovering over the Badge.
-   */
   iconColorHovered?: ColorValue;
-
-  /**
-   * The icon color when the Badge is being pressed.
-   */
   iconColorPressed?: ColorValue;
-
-  /**
-   * The size of the icon.
-   */
   iconSize?: number;
-
-  /**
-   * The weight of the lines used when drawing the icon.
-   */
   iconWeight?: number;
-
-  /**
-   * The height of the Badge.
-   */
   height?: ViewStyle['height'];
-
-  /**
-   * The width of the Badge.
-   */
   width?: ViewStyle['width'];
+  top?: NativeScrollRectangle['top'];
```

Comment:
I would've expected top/left/bottom/right to be ViewStyle properties. I'm not sur ewhat NativeScrollRectangle is, but given that Badge doesn't have to do with scroll, it seems like the wrong type to be pulling this from?

Reply:
Also, can we add comments on what this is for, and reorder in alphabetical order, please?

Reply:
I'm also not seeing top or left used/passed around anywhere, did I miss it?

Reply:
> I'm also not seeing top or left used/passed around anywhere, did I miss it?

I'll will add it, so customers would be able to customize

Reply:
> Also, can we add comments on what this is for, and reorder in alphabetical order, please?

Should all of it be in alphabetical order? I was thinking about splitting it in groups, for example, "top, left, right and bottom" is one group, shapes and sizes would be other groups. But to have alphabetical order inside groups

Reply:
> > I would've expected top/left/bottom/right to be ViewStyle properties. I'm not sur ewhat NativeScrollRectangle is, but given that Badge doesn't have to do with scroll, it seems like the wrong type to be pulling this from?
> 
> ViewStyle doesn't contain top, left, right and bottom

It's under FlexStyle (which ViewStyle extends) according to DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/a574bec6112488893ca55fe25cec84b9ddfaac2f/types/react-native/index.d.ts#L687


> > Also, can we add comments on what this is for, and reorder in alphabetical order, please?
> 
> Should all of it be in alphabetical order? I was thinking about splitting it in groups, for example, "top, left, right and bottom" is one group, shapes and sizes would be other groups. But to have alphabetical order inside groups

I've been ordering everything in alphabetical order so far as that's how RN documentation organizes their props. I think it'd be good to have consistency, but if you'd like to group them I think it'd make sense to add some sort of header comment to label the groups.